data/config.dist.json

```diff
@@ -5,6 +5,6 @@
     "array", "string", "int", "float", "bool", "iterable", "callable", "void"
   ],
   "php-core-extensions" : [
-    "Core", "date", "pcre", "Reflection", "SPL", "standard"
+    "Core", "date", "pcre", "Phar", "Reflection", "SPL", "standard"
```

Comment:
Sure. If @keradus would split the line, that would be great.
@keradus may you please adjust the default config here, so the test passes again?
https://github.com/maglnet/ComposerRequireChecker/blob/master/src/ComposerRequireChecker/Cli/Options.php